lib/inspec/metadata.rb

```diff
@@ -35,14 +35,9 @@ def initialize(ref, logger = nil)
     end
 
     def supports(sth, version = nil)
-      params[:supports] ||= []
```

Comment:
we share the same implementation for `inspec.yml` and `metadata.rb`. I would assume, this breaks the new feature as well?